modules/shared/components/atoms/Radio/IRadio.d.ts

```diff
@@ -1,15 +1,22 @@
 import type React from 'react';
 
 declare namespace IRadio {
-  export interface IProps {
+  export interface IProps extends IData, IHandlers {
     size: 'default' | 'small';
     name: string;
-    value: string;
-    id: string;
     defaultChecked: boolean;
     disabled?: boolean;
     onlyLabel?: boolean;
+  }
+
+  export interface IData {
+    value: string;
+    id: string;
+  }
+
+  export interface IHandlers {
```

Comment:
- what is the gain from creating two extra interfaces `IDate, IHandlers` 🤔?
- are not all of them are considered `interface for the component props`.
- which rule you followed to separate these fields., Ex: why you did not include the `name` in the `IData` interface

Reply:
Please join the standup any day if you have time so we can explain what happened in the last 4 or 5 days better than discussing it here